### DISCUSSION TITLE
Inaccessible host

### DISCUSSION THREAD

**Discussion by shreyansh-24** [cloud-carbon-footprint/cloud-carbon-footprint#1062] [General]
error: Unable to process footprint request.Error: Athena start query failed. Reason Inaccessible host: `athena.mumbai.amazonaws.com' at port `undefined'. This service may not be available in the `mumbai' region..

Is the service not available in mumbai or am i making some mistake in integration. I followed guided install route

**Comment by ericksod** [comment]
Hi @shreyansh-24!
Can you tell us what environment variables you have set? 
And have you fully configured your AWS credentials using the AWS CLI?

**Comment by 4upz** [comment]
Closing this issue since follow-up conversations have insinuated that this was fixed. @shreyansh-24 If there was something that you've done to fix this issue, feel free to share! I'm also happy to reopen this discussion if it's persisting and you need additional assistance
